cmd/root.go

```diff
@@ -14,9 +15,27 @@ var rootCmd = &cobra.Command{
 	Long:  "A plugin that makes using Spin with OTel easy by automatically standing up dependencies, sourcing environment variables, and linking to dashboards.",
 }
 
-var otelConfigDir = "otel-config"
+var otelConfigDirName = "otel-config"
+var otelConfigPath string
+
+// setOtelConfigPath allows for someone to run the otel plugin directly from source or via the Spin plugin installation
+func setOtelConfigPath() error {
+	executablePath, err := os.Executable()
+	if err != nil {
+		return err
+	}
+
+	otelConfigPath = path.Join(path.Dir(executablePath), otelConfigDirName)
+
+	return nil
+}
 
 func Execute() {
+	if err := setOtelConfigPath(); err != nil {
```

Comment:
is it idiomatic `go` to combine these this way? I would expect it to be two different statements.

Reply:
Do you mean like this instead?
```go
err := setOtelConfigPath();
if err != nil {
  return err
}
```
I think it's an issue of preference because both work.

Reply:
yes, I find this version much more readable but again take anything I say with a mountain of salt since I do not know idiomatic go.

Reply:
@calebschoepp Thoughts?

Reply:
What Andrew has done here is a pretty common Go pattern, I'm okay with it.